.circleci/values/oathkeeper-maester.yaml

```diff
@@ -0,0 +1,2 @@
+---
+# singleNamespaceMode: true
```

Comment:
why do we have it here commented out?

Reply:
we don't want it to be set by default, but having this here makes debugging easier, as it is enough to uncomment it and use the file :)